ECMAScript 5.1 is not available by default

dilyanpalauzov opened the pull request.

rkoshak commented on tutorials/getting_started/rules_blockly.md:
```diff
@@ -11,7 +11,7 @@ Both of these are a way for you to write code that can be as long and complex as
 As you recall from the previous page, one of the four types of conditions are Script Conditions and one of the four types of actions are Script Actions.
 Choosing either of those presents you with a choice of language to use.
-By default there are only three choices: Blockly, ECMAScript, and Rules DSL.
+By default there are only two choices: Blockly and Rules DSL.
```
Blockly requires JS Scripting so really Rules DSL is the only default

rkoshak commented on tutorials/getting_started/rules_introduction.md:
```diff
@@ -106,7 +106,7 @@ There is a growing list of options of ways for users to create rules and many of
 See the docs for the chosen language for details on how to write rules in files for that language.
 
-openHAB comes with three languages by default (Rules DSL, ECMAScript 5.1, and Blockly) and has a number of [automation add-ons](/addons/#automation) to add more.
+openHAB comes with two languages by default (Rules DSL and Blockly) and has a number of [automation add-ons](/addons/#automation) to add more.
```
openHAB comes with one language by default (Rules DSL) and has ...

rkoshak reviewed (commented):
Blockly depends on the JS Scripting add-on (since OH 4) so Rules DSL is the only default.

dilyanpalauzov commented:
As tutorials/getting_started/rules_introduction.md contains “For this tutorial we will be using Blockly, which requires installing the JavaScript Scripting add-on.”, I do not repeat the same information at tutorials/getting_started/rules_blockly.md .

There are in fact ten types of Conditions shown in MainUI, if one clicks on “Show all”.  So I have removed the number of available conditions.

Likewise, there are nine type of actions (when mqtt is installed), so I have removed the number of available actions.

“Script Action” means selecting in the UI “Inline Script”.  I have replaced once “Script Action” with “Inline Script”, leaving “Script Action” on other places.
